### PR TITLE
Read secret key from the environment

### PR DESCRIPTION
**Pull request by krzysztofwolski**

### CODE REVIEW

File: saleor-app.ts [hunk 1]
```diff
@@ -40,6 +40,14 @@ switch (aplType) {
     throw new Error("Invalid APL config, ");
   }
 }
+
+if (!process.env.SECRET_KEY && process.env.NODE_ENV === "production") {
```
Comment (lkostrowski) [inline saleor-app.ts:44]: its fine but it should be a part of app-sdk

**Review by lkostrowski — APPROVED** [review]

**Review by lkostrowski — APPROVED** [review]

File: saleor-app.ts [hunk 2]
```diff
@@ -40,6 +40,14 @@ switch (aplType) {
...
+  throw new Error(
+    "For production deployment SECRET_KEY is mandatory to use EncryptedSettingsManager.",
+  );
+}
+export const settingsManagerSecretKey = process.env.SECRET_KEY || "development";
```
Comment (lkostrowski) [inline saleor-app.ts:49]: `development` -> `CHANGE_ME`

File: saleor-app.ts (hunk 1 above)
Comment (krzysztofwolski) [reply saleor-app.ts:44]: We do not _need_, but this code will fail at startup time instead of first metadata request

Comment (lkostrowski) [reply saleor-app.ts:44]: Ok lets keep it

**Review by lkostrowski — APPROVED** [review]

**Pull request merged**